content/how-to/fabric/fabric-ticketing/index.md

```diff
@@ -0,0 +1,153 @@
+
+### Global Dashboard Overview
+
+<img width="945" alt="FT6" src="https://user-images.githubusercontent.com/98875667/156172786-71f7f0d1-5193-4fc4-a8f3-e4ef9b35a8b9.png">
+
+For all multi-cloud and non-dedicated users, the ticket navigation option will take you to the new ticketing application as shown in the above image.
+
+The global dashboard view has tabs as listed below:
+* Tickets
+* Support
+* Billing
+* Account
+
+#### Tickets
+
+<img width="177" alt="FT4" src="https://user-images.githubusercontent.com/98875667/156175991-029922e0-4e20-4b08-af23-1088e2e62d1b.png">
+
+The user must be granted ticketing permissions for the ticket navigation items to be allowed.
+
+### Application Overview
+
+<img width="950" alt="FT7" src="https://user-images.githubusercontent.com/98875667/156180814-ee83a84d-ece0-41ff-b6fd-4e1019a99ad6.png">
+
+**Ticket List Page**: Displays the list of tickets created. The list displays either the most recently created ticket or last modified ticket at the top. 
+
+#### **Left Side Ticket Panel**: 
+
+<img width="302" alt="FT9" src="https://user-images.githubusercontent.com/98875667/156182984-cf86867f-a906-45bd-9e37-e0a8daba6c67.png">
+
+1. Create Ticket: User can create new ticket using this option. Details of how new ticket is created is detailed in upcoming sections. 
+2. Search: User can search the required tickets. 
+
+<img width="142" alt="FT10" src="https://user-images.githubusercontent.com/98875667/156183312-0c15933a-b591-4deb-8dd8-1ccbb2682681.png">
+
+3. Status Drop Down: Allows a user to narrow down the list results by ticket status of Open Tickets, Pending Customer, Closed Tickets, and All Tickets. 
+
+<img width="257" alt="FT11" src="https://user-images.githubusercontent.com/98875667/156183533-ce5b707d-2d71-45ea-b626-2c5c1ec7c9e4.png">
+
+4. Filter: This filter option can filter your tickets using options such as Device, Created By, Accounts, and Date Range 
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list. 
+
+#### **Right Side Ticket Panel**:  
+
+**Selected Ticket Panel**: Displays the details of the selected ticket. This loads the first ticket list item by default. 
+
+<img width="621" alt="FT13" src="https://user-images.githubusercontent.com/98875667/156184075-2b34647c-1c4d-4053-b4ea-20486b52b52f.png">
+
+<img width="374" alt="FT8" src="https://user-images.githubusercontent.com/98875667/156186872-21df8151-9e8f-44f2-abb4-c9fca88e1902.png">
+
+You can load your ticket in a new browser tab by selecting the ticket ID from the header of the right ticketing panel as shown in above image.
+
+The right-side ticket panel displays the details of the selected ticket. The details of the ticket are divided into three tabs such as: 
+
+* **Comments**: Comment option displays the details of the most recently created ticket. 
```

Comment:
should be `Comment option displays all comments for the selected ticket.`

Reply:
Updated as suggested